src/components/card/card.pw.tsx

```diff
@@ -0,0 +1,353 @@
+    );
+  });
+
+  test("should allow custom boxShadow and hoverBoxShadow prop values and they apply the expected styling", async ({
+    mount,
+    page,
+  }) => {
+    await mount(
+      <CardComponent
+        boxShadow="boxShadow400"
+        hoverBoxShadow="boxShadow200"
+        interactive
+      />
+    );
+    const cardElement = await card(page);
+    await expect(cardElement).toHaveCSS(
+      "box-shadow",
+      "rgba(0, 20, 30, 0.04) 0px 10px 40px 0px, rgba(0, 20, 30, 0.1) 0px 50px 80px 0px"
+    );
+    await cardElement.hover();
+    await cardElement.waitFor();
+
+    await expect(cardElement).toHaveCSS("cursor", "pointer");
+    await expect(cardElement).toHaveCSS(
+      "box-shadow",
+      "rgba(0, 20, 30, 0.2) 0px 10px 20px 0px, rgba(0, 20, 30, 0.1) 0px 20px 40px 0px"
+    );
+  });
+
+  textAlignment.forEach((align) => {
+    test(`should check ${align} alignment applies the expected styling`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<CardTextAlignment align={align} />);
+
+      const columnCardElement = await columnCard(page);
+
+      await expect(columnCardElement).toHaveCSS("text-align", align);
+    });
+  });
+
+  ([375, 535, 777] as const).forEach((height) => {
+    test(`should check ${height} height is applied`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<CardComponent height={`${height}px`} />);
+
+      const cardElement = await card(page);
+
+      await assertCssValueIsApproximately(cardElement, "height", height);
+    });
+  });
+
+  test("should call onClick callback when a click event is triggered", async ({
+    mount,
+    page,
+  }) => {
+    let setClickCounter = 0;
+    await mount(
+      <CardComponent
+        interactive
+        action={() => {
+          setClickCounter += 1;
+        }}
+      />
+    );
+
+    const cardElement = await card(page);
+
+    await cardElement.click({
+      force: true,
+    });
+
+    await cardElement.waitFor();
+    expect(setClickCounter).toEqual(1);
+  });
+});
+
+(["default", "large"] as CardProps["roundness"][]).forEach((roundness) => {
+  test(`Should have the expected border radius styling when roundness is ${roundness}`, async ({
+    mount,
+    page,
+  }) => {
+    const result = roundness === "default" ? "8px" : "16px";
+
+    await mount(<CardComponent roundness={roundness} />);
+
+    const cardElement = await card(page);
+
+    await expect(cardElement).toHaveCSS("border-radius", result);
+
+    const footerCardElement = await footerCard(page);
+
+    await expect(footerCardElement).toHaveCSS(
+      "border-radius",
+      `0px 0px ${result} ${result}`
+    );
+  });
+});
+
+test.describe("Accessibility tests for Card component", () => {
+  test("should pass accessibility tests for default example", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<CardComponent />);
+    await checkAccessibility(page);
```

Comment:
I think there should be a space between the mount and the test for the accessibility tests to make them follow the same pattern as the other tests in the file and of other components.

Reply:
Will do. This is the kind of thing I figured the linter would do. Especially if we do have this as a pattern.